server.js

```diff
@@ -17,6 +12,65 @@ mongoose.Promise = Promise;
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
+
+const Song = mongoose.model("Song", {
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+
+})
+// The reset of database and adding new songs or albums 
+// was great for educational purpose but is not to be used live.
+//
+// const Album = mongoose.model("Album", {
+//   title: String,
+//   tracks: Number,
+//   song: String
+// })
+
+if(process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach(singleSong => {
+      const newSong = new Song(singleSong);
+      newSong.save();
+    })
+    const testSong = new Song ({ 
+      id: 75,
+      trackName: 'TestSong',
+      artistName: 'TestArtist',
+      genre: 'TestGenre',
+      bpm: 190,
+      energy: 70,
+      danceability: 45,
+      loudness: 90,
+      liveness: 80,
+      valence: 34,
+      length: 937,
+      acousticness: 23,
+      speechiness: 46,
+      popularity: 97
+     })
+    await testSong.save()
```

Comment:
This is good as a practice, but you might not need this anymore since your endpoints work :)